SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a security issue, please email the maintainers (see
+[AUTHORS](https://github.com/liblouis/liblouis/blob/master/AUTHORS))
+with a description of the issue, the steps you took to create the
+issue, affected versions, and, if known, mitigations for the issue.
+
+Our vulnerability management team will respond within 3 working days
+of your email. If the issue is confirmed as a vulnerability, we will
+open a Security Advisory. This project follows a 90 day disclosure
+timeline.
```

Comment:
What does this mean in practice?

Reply:
There are a number of things we have to do when e get a security report:

1. confirm that we got the mail. The template says "3 working days" but I find that a bit too strict. The Debian policy just says "several days at most". Maybe we could say 5 working days?
2. After some time the reporter is allowed to publicly announce the vulnerability. During that time we have time to quietly fix the problem without anyone else knowing about it. Here it is defined as 90 days.

Reply:
Sounds good.